server/game/cards/09.1-FtE/FalseLoyalties.js

```diff
@@ -0,0 +1,42 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+const AbilityDsl = require('../../abilitydsl');
+
+class FalseLoyalties extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            title:'Switch 2 characters your opponent controls',
+            when: {
+                afterConflict: (event, context) => {
+                    return event.conflict.winner === context.player.opponent &&
+                    context.player.honor < context.player.opponent.honor;
```

Comment:
This will give a null reference exception in single player can you please adjust to:

```js
return context.player.opponent && event.conflict.winner === context.player.opponent &&
    context.player.honor < context.player.opponent.honor;
```